js/scripts/util.js

```diff
@@ -36,11 +36,6 @@ const PHYSICS = {
     TERMINAL_VELOCITY : 200 // currently only being applied to projectiles
 };
 
-const FONT = { // NOTE: the "vt323" part below is not what's assigning the vt323 font. It's the CSS file. I kept entity here for reference.
```

Comment:
Dialog relies on this constant, let's add this back.

Reply:
It will be added back when my pull request is approved!

Reply:
Great, thanks for letting me know!